controllers/configurationpolicy_controller.go

```diff
@@ -96,6 +96,7 @@ type discoveryInfo struct {
 	apiResourceList        []*metav1.APIResourceList
 	apiGroups              []*restmapper.APIGroupResources
 	discoveryLastRefreshed time.Time
+	refreshError           error
```

Comment:
What's the purpose for this being a struct member as opposed to a local variable? I don't see it used outside of the `refreshDiscoveryInfo` method.